Dockerfile

```diff
@@ -21,16 +21,12 @@ RUN make
 
 FROM alpine:3.17 as kubearmor
 
-RUN apk --no-cache update
 RUN echo "@community http://dl-cdn.alpinelinux.org/alpine/edge/community" | tee -a /etc/apk/repositories
-RUN echo "@testing http://dl-cdn.alpinelinux.org/alpine/edge/testing" | tee -a /etc/apk/repositories
 
 RUN apk --no-cache update
-RUN apk add bash curl procps
```

Comment:
Folks, do we need procps? Here are the package contents: https://pkgs.alpinelinux.org/contents?branch=v3.17&name=procps&arch=x86_64&repo=main

Reply:
@daemon1024 pointed out this was used earlier when the entrypoint was a bash script and we used the package for handling signals. We don't need it now.